tests/app/template_statistics/test_rest.py

```diff
@@ -189,3 +191,42 @@ def test_get_template_statistics_for_template_returns_empty_for_old_notification
     )
 
     assert not json_resp['data']
+
+
+def test_get_last_used_datetime_for_template(
+    admin_request, sample_template
+):
+    date_from_notification = datetime.utcnow() - timedelta(hours=2)
+    create_notification(template=sample_template, created_at=date_from_notification)
+    date_from_ft_status = (datetime.utcnow() - timedelta(days=2)).date()
+    create_ft_notification_status(bst_date=date_from_ft_status,
+                                  template=sample_template)
+
+    json_resp = admin_request.get(
+        'template_statistics.get_last_used_datetime_for_template',
+        service_id=str(sample_template.service_id),
+        template_id=sample_template.id
+    )
+    assert json_resp['last_date_used'] == date_from_notification.strftime(DATETIME_FORMAT)
+
+
+def test_get_last_used_datetime_for_template_returns_400_if_service_does_not_exist(
```

Comment:
minor, but this and the function below should be `...returns_404_if....`, not returns 400